cli/internal/core/engine.go

```diff
@@ -58,24 +67,6 @@ type EngineBuildingOptions struct {
 	TasksOnly bool
 }
 
-// Prepare constructs the Task Graph for a list of packages and tasks
-func (e *Engine) Prepare(options *EngineBuildingOptions) error {
```

Comment:
Renamed `generateTaskGraph()` to `Prepare()` , since this is an empty shell of a function now.